dashboard/public/pixelation/pixelation.js

```diff
@@ -604,20 +662,30 @@ function getPositiveValue(element) {
   return value >= 1 ? value : 1;
 }
 
-function updateBinaryDataToMatchSliders() {
+/**
+ * Gets the numbers stored in the slider and returns them as a byte string.
+ */
+function getSliderBytes() {
   var heightByte = pad(getPositiveValue(heightRange).toString(2), 8, "0");
   var widthByte = pad(getPositiveValue(widthRange).toString(2), 8, "0");
   var bppByte = pad(getPositiveValue(bitsPerPixelRange).toString(2), 8, "0");
 
-  var justBits = pixel_data.value.replace(/[ \n]/g, "");
+  var sliderBits = widthByte + heightByte;
+  if (options.version === "3") {
+    sliderBits += bppByte;
```

Comment:
what's bpp? Can this have a more descriptive name?

Reply:
Fixed!